sources/specUtils.ts

```diff
@@ -44,6 +44,8 @@ export async function findProjectSpec(initialCwd: string, locator: Locator, {tra
   // A locator is a valid descriptor (but not the other way around)
   const fallbackLocator = {name: locator.name, range: locator.reference};
 
+  if (process.env.COREPACK_ENABLE_STRICT === `0`) return fallbackLocator;
```

Comment:
Nit: might not be needed in this PR, but it'd make sense to move the "parsing" logic in a function so we could be sure the same one is used for enableStrict and enableNetwork